magprime/automated_emails.py

```diff
@@ -34,7 +34,7 @@ def __init__(self, event):
                sender='MAGFest LAN <lan@magfest.org>')
 
 AutomatedEmail(Attendee, 'MAGFest Tech Ops volunteering', 'techops.txt',
-           lambda a: a.staffing and a.requested(c.TECH_OPS) and not a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>')
+           lambda a: a.staffing and a.requested(c.TECH_OPS) and not a.assigned_to(c.TECH_OPS), sender='MAGFest TechOps <techops-heads@magfest.org>' 'cc=techops-heads@magfest.org')
```

Comment:
This needs to read ``, cc='techops-heads@magfest.org'`` instead of `` 'cc=techops-heads@magfest.org'``.  In other words, the quotes need to go around the email address but not the ``cc=``, and there needs to be a comma after the previous argument.